components/brave_wallet/browser/brave_wallet_p3a.cc

```diff
@@ -180,8 +180,6 @@ void BraveWalletP3A::ReportJSProvider(mojom::JSProviderType provider_type,
                      : JSProviderAnswer::kNativeNotOverridden;
       }
       break;
-    default:
-      NOTREACHED_NORETURN();
```

Comment:
shouldn't this also early return by default? same as two mojom::CoinType below.

Reply:
following the suggestion above: https://github.com/brave/brave-core/pull/26247#discussion_r1817420831

Reply:
I agree with that, what about the other two return in this file?

Reply:
oh, I see. we only choose two from that type for first one and another one is else condition